edge/pkg/eventbus/event_bus.go

```diff
@@ -3,7 +3,6 @@ package eventbus
 import (
 	"encoding/json"
 	"fmt"
-
```

Comment:
nit: Please don't remove this blank line. It is used to seperate golang and vendor packages

Reply:
OK